pages/Months.jsx

```diff
@@ -0,0 +1,62 @@
+import React, { useEffect, useState } from 'react'
+import Month from '../components/Month'
+import Search from '../components/Search'
+import Title from '../components/Title'
+import Page from '../components/Page'
+import { filterMonths, retrieveMonths } from '../utils/months'
+// console.log('retrieveMonths: ', retrieveMonths);
+
+export default () => {
+  const [monthsList, setMonthsList] = useState([])
+  // console.log('useState: ', useState);
+  // console.log('monthsList: ', monthsList);
+  const [filteredMonthsList, setFilteredMonthsList] = useState([])
+  const [searchTerm, setSearchTerm] = useState('')
+
+  useEffect(() => {
+    // console.log('useEffect: ', useEffect);
+    async function pullData() {
+      // console.log('pullData: ', pullData);
+      const months = await retrieveMonths()
+      // console.log('months: ', months);
+
+      setMonthsList(months)
+      setFilteredMonthsList(months)
+      // console.log('months: ', months)
+      // console.log('setFilteredMonthsList: ', setFilteredMonthsList);
```

Comment:
Never leave commented code in the code base. It should always be cleaned up prior to making a PR